### PR TITLE
Update Nosto models to convert currencies into default display currency

### PR DESCRIPTION
**Pull request by hannunosto**
Also change export sort order to be created desc

### CODE REVIEW

File: app/code/community/Nosto/Tagging/Model/Meta/Order.php [hunk 1]
```diff
@@ -163,8 +166,8 @@ public function loadData(Mage_Sales_Model_Order $order)
                         'productId' => -1,
                         'quantity' => 1,
                         'name' => 'Shipping and handling',
-                        'unitPrice' => new NostoPrice($shippingInclTax),
-                        'currency' => new NostoCurrencyCode($order->getBaseCurrencyCode())
+                        'unitPrice' => new NostoPrice($priceHelper->convertToDefaultCurrency($shippingInclTax, $store)),
```
Comment (mridang) [inline app/code/community/Nosto/Tagging/Model/Meta/Order.php]: Split into declaration and assignment like L198.

File: app/code/community/Nosto/Tagging/Model/Meta/Order.php [hunk 2]
```diff
@@ -150,8 +153,8 @@ public function loadData(Mage_Sales_Model_Order $order)
                         'productId' => -1,
                         'quantity' => 1,
                         'name' => 'Discount',
-                        'unitPrice' => new NostoPrice($discount),
-                        'currency' => new NostoCurrencyCode($order->getBaseCurrencyCode())
+                        'unitPrice' => new NostoPrice($priceHelper->convertToDefaultCurrency($discount, $store)),
```
Comment (mridang) [inline app/code/community/Nosto/Tagging/Model/Meta/Order.php]: Split into declaration and assignment like L198.

File: app/design/frontend/base/default/template/nostotagging/cart.phtml [hunk 3]
```diff
@@ -36,8 +36,9 @@ $cart = $this->getCart();
 $helper = Mage::helper('nosto_tagging');
 $priceFormatter = Nosto::formatter('price');
 $priceFormat = new NostoPriceFormat(2, '.', '');
+$store = Mage::app()->getStore();
```
Comment (mridang) [inline app/design/frontend/base/default/template/nostotagging/cart.phtml]: List all cart items in the current currency. No need to convert to default anymore.

**Comment by hannunosto** [comment]
This PR is outdated. Fixes implemented in feature/price-variants-api

**Pull request closed without merging**